sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -398,6 +400,18 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class Delay
+    {
+        protected Delay(System.TimeSpan? maxDelay = default(System.TimeSpan?), double jitterFactor = 0.2) { }
+        public static Azure.Core.Delay CreateExponentialDelay(System.TimeSpan? initialDelay = default(System.TimeSpan?), System.TimeSpan? maxDelay = default(System.TimeSpan?)) { throw null; }
+        public static Azure.Core.Delay CreateFixedDelay(System.TimeSpan? delay = default(System.TimeSpan?)) { throw null; }
+        public System.TimeSpan GetNextDelay(Azure.Response? response, int retryNumber, System.TimeSpan? serverDelayHint, System.Collections.Generic.IDictionary<string, object?> context) { throw null; }
+        public System.Threading.Tasks.ValueTask<System.TimeSpan> GetNextDelayAsync(Azure.Response? response, int retryNumber, System.TimeSpan? serverDelayHint, System.Collections.Generic.IDictionary<string, object?> context) { throw null; }
+        protected abstract System.TimeSpan GetNextDelayCore(Azure.Response? response, int retryNumber, System.Collections.Generic.IDictionary<string, object?> context);
+        protected abstract System.Threading.Tasks.ValueTask<System.TimeSpan> GetNextDelayCoreAsync(Azure.Response? response, int retryNumber, System.Collections.Generic.IDictionary<string, object?> context);
+        protected static System.TimeSpan Max(System.TimeSpan t1, System.TimeSpan t2) { throw null; }
```

Comment:
Curious if there's a way to move these out of the public API.  They feel like TimeSpan utilities that could be used more broadly than Delay, and might better live in internal shared source?

Reply:
Is there a way to provide them without putting them on the public API?  Are there other places in Core where it might be useful to use the same logic to compare TimeSpans?

Reply:
Well it depends on what the goal is. If we want customers to have these conveniences (which was my intent) then it would need to be a public API.

Reply:
Well, I don't think we would end up releasing public utility functions for this, so was hoping to sneak it in here 😆 

I'm not sure if there is a specific guideline here, but am okay with removing them from the public API if that is the consensus.